README.md

```diff
@@ -42,6 +46,225 @@ nm.init()
 });
 ```
 
-## References
+## HTTP APIs
+
+### Connect to network
+
+POST `/connect-network`
+
+Body:
```

Comment:
Should this also mention the post data encoding?

Reply:
Good call